test/powershell/engine/ExecutionPolicy.GroupPolicy.Tests.ps1

```diff
@@ -0,0 +1,48 @@
+﻿##
+## PowerShell ExecutionPolicy GroupPolicy tests
+## These are Windows only tests
+##
+
+Describe "User group policy execution policy should work" -Tags 'Feature' {
```

Comment:
I don't think we have such a tag yet, but I think we need a tag saying the test changes important machine state.

I wouldn't want to run this test on my dev machine, especially knowing that `AfterAll` isn't guaranteed to run, e.g. if I hit Ctrl+C. My machine will be left in a weird state that would be hard to track down.

On a throw-away test machine like in CI - that's a non-issue of course.